docs/tutorials/infer_phys_pars.rst

```diff
@@ -466,29 +469,31 @@ objects correctly when using the small-angle approximation
 
     s = [0., 1.]
 
-    v_p_sys_eff = ((d_eff * (mu_alpha_star * np.sin(xi) + mu_delta * np.cos(xi)))
-                   .to(u.km/u.s, equivalencies=u.dimensionless_angles()))
+    mu_p_sys = psr_coord.pm_ra_cosdec * np.sin(xi) + psr_coord.pm_dec * np.cos(xi)
 
-    v_lens1 = s * (np.sqrt(d_eff) *  dveff_c + v_p_sys_eff)
-    v_lens2 = s * (np.sqrt(d_eff) * -dveff_c + v_p_sys_eff)
+    v_eff_p_sys = ((d_eff * mu_p_sys)
+                   .to(u.km/u.s, equivalencies=u.dimensionless_angles()))
+    
+    v_lens = s * (v_eff_p_sys + [[+1.], [-1.]] * np.sqrt(d_eff) * dveff_c)
 
 Because only the *norm* of the scintillation velocity can be measured, there
 are two possible solutions for :math:`v_\mathrm{lens}`: one in which the lens
 motion and the pulsar's systemic motion add up to a large offset in
 scintillation velocity, and one in which they counteract one another's
 contribution to the scintillation-velocity offset. For known values of the
-scaled-scintillation-velocity offset :math:`C` and the pulsar's systemic
-effective velocity :math:`v_\mathrm{p,sys,eff}`, this translates to solutions
-for :math:`v_\mathrm{lens}` with low and high absolute values, respectively.
+scaled-scintillation-velocity offset :math:`C` and the pulsar's systemic-motion
+contribution to the effective velocity :math:`v_\mathrm{eff,p,sys}`, this
+translates to solutions for :math:`v_\mathrm{lens}` with low and high absolute
+values, respectively.
 
 .. jupyter-execute::
 
     plt.figure(figsize=(7., 6.))
 
-    plt.plot(s, v_lens1.to(u.km/u.s),
+    plt.plot(s, v_lens[0,:].to(u.km/u.s),
```

Comment:
I guess here you have the two different labels, which may make it more difficult to combine into one plot command.

Reply:
I put the labels in ``legend``.